source/guide_drupal.rst

```diff
@@ -125,30 +124,49 @@ Instead, we install it next to that and then use a symbolic link to make it acce
 RewriteBase
 -----------
 
-Search for ``# RewriteBase /`` and remove the leading ``#`` to activate this statement.
+Edit ``/var/www/virtual/isabell/drupal/web/.htaccess``, search for ``# RewriteBase /`` and remove the leading ``#`` to activate this statement.
 
 ::
 
-  [isabell@stardust isabell]$ vi drupal/web/.htaccess
   RewriteBase /
 
 
 SCM
 ---
 
-It is a very good time to initialise a repository witin the ``drupal`` directory.
+It is a very good time to initialise a repository within the ``drupal`` directory.
 This step is optional but highly recommended ;-D 
 
-drush-launcher
---------------
+drush
+-----
+
+In order to make use  of Drupal Composer Template's drush_, 
+you will add the path of your Drupal Composer Template's bin directory, 
+which contains drush´s executable to the PATH environment variable. 
+It will detect *DRUPAL_ROOT*.
+
+Place the following statement to the startup file of your shell, e.g. ``~/.bashrc`` or ``~/.zshrc``:
+
+::
 
-In order to make use  of `Drupal Composer Template`_'s drush, you will need to install the `drush-launcher`_:
+  export PATH=$PATH:/var/www/virtual/$USER/drupal/vendor/bin
```

Comment:
Are there multiple binaries here or just `drush`? In case it's just drush, a `ln -s /var/www/virtual/$USER/drupal/vendor/bin/drush ~/bin` might be quicker and more resilient.

Reply:
Actually there are multiple binaries, like the latest version of _DrupalConsole_ and _phpcs_, _phpunit_, _phpcbf_ , _robo_ and others which come with this specific [drupal-composer template](https://github.com/drupal-composer/drupal-project).